apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant/Defendant.tsx

```diff
@@ -452,7 +454,9 @@ const Defendant = () => {
     }))
   }
 
-  const stepIsValid = isDefendantStepValidIndictments(workingCase)
+  const stepIsValid =
+    isDefendantStepValidIndictments(workingCase) &&
+    Boolean(workingCase.id || isProsecutorSelected)
```

Comment:
Why checking if `workingCase.id` is defined? isn't `isProsecutorSelected` sufficient? or is the assumption if a case id exists a prosecutor must also exist?

Reply:
Discuessed irl. I've added a comment for clarity.